ooiui/static/js/views/asset_management/AssetView.js

```diff
@@ -6,13 +6,20 @@ var ParentAssetView = Backbone.View.extend({
      *  - derender()
      */
     initialize: function() {
-        _.bindAll(this, 'render', 'derender');
+        "use strict";
```

Comment:
@birdage but that's F**** S****...so lets move it to the top and only do it once.  LOL